### DISCUSSION TITLE
The default behavior of "protocol: dns"

### DISCUSSION THREAD

**Discussion by Tigerfyj** [shadowsocks/shadowsocks-rust#547] [Q&A]
I would like to know how to use the "protocol: dns".
Q1: The readme says it depends on ACL rules. But, do we have any built-in rules within the binary?
Q2: If not, what is the default behavior? Does it always go to the remote dns server?
Q3: I didn't find an option for ACL file in the config file. Does it have to be a cmd flag?

Thanks for your help!

**Comment by zonyitoo** [comment]
Q1: No.
Q2: Yes. All goes to the remote.
Q3: `--acl`
